di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/config/OrchestratorConfig.java

```diff
@@ -9,8 +9,6 @@ public class OrchestratorConfig {
                     "IPV_BACKCHANNEL_ENDPOINT", "https://api.identity.build.account.gov.uk/");
     public static final String IPV_BACKCHANNEL_TOKEN_PATH =
             getConfigValue("IPV_BACKCHANNEL_TOKEN_PATH", "/dev/token");
-    public static final String IPV_BACKCHANNEL_USER_IDENTITY_PATH =
-            getConfigValue("IPV_BACKCHANNEL_USER_IDENTITY_PATH", "/dev/user-identity");
```

Comment:
The removal of this config from common-infra repos is addressed in this ticket: https://govukverify.atlassian.net/browse/PYIC-6568